src/App.tsx

```diff
@@ -19,7 +19,6 @@ import { getDashboardAppBasePath } from "./utils";
 
```

Comment:
add test in PR checklist for testing assigning / managing roles for a user who is a part of multiple tenants.

Reply:
will do.